pkg/services/publicdashboards/metric/metrics.go

```diff
@@ -0,0 +1,23 @@
+package metric
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+)
+
+var (
+	namespace = "grafana"
+)
+
+type Metrics struct {
+	PublicDashboardsTotal *prometheus.GaugeVec
+}
+
+func NewMetrics() *Metrics {
+	return &Metrics{
+		PublicDashboardsTotal: prometheus.NewGaugeVec(prometheus.GaugeOpts{
+			Namespace: namespace,
+			Name:      "public_dashboards_total",
```

Comment:
I don't think you want this to end in `_total` since it's not strictly cumulative?

> Note that an accumulating count has `total` as a suffix, in addition to the unit if applicable.

from https://prometheus.io/docs/practices/naming/

Reply:
public_dashboards_amount maybe?

Reply:
Yeah, that could work :+1: